sdk/test/logs/BUILD

```diff
@@ -17,6 +17,8 @@ cc_test(
     ],
     deps = [
         "//sdk/src/logs",
+        "//sdk/src/trace",
+        "//exporters/memory:in_memory_span_exporter",
```

Comment:
unsure why this is needed - will take a look at unit tests to see if this can be removed